src/docs/asciidoc/reference/configuration/user-management-properties.adoc

```diff
@@ -0,0 +1,245 @@
+      }
+    }
+  }
+}
+----
+
+.Mandatory property
+[NOTE]
+====
+This property is mandatory.
+====
+
+.URL Redirection
+
+When an end-user validates his account, he is redirected to a specific URL of your application. There are 2 cases, if the confirmation is OK or if it failed:
+
+[source, json]
+----
+{
+  "registration": {
+    "confirmation": {
+      "success-url": "https://my-app-url/#login",                   // <1>
+      "failure-url": "https://my-app-url/#confirmation-failed"      // <2>
+    }
+  }
+}
+----
+<1> The end-user is redirected to this URL when the confirmation is OK
+<2> The end-user is redirected to this URL when the confirmation failed
+
+
+You can also only set the path of your urls. The base url will be automatically provided by ZetaPush:
+
+[source, json]
+----
+{
+  "registration": {
+    "confirmation": {
+      "success-url": "#login",
+      "failure-url": "#confirmation-failed"
+    }
+  }
+}
+----
+
+.Mandatory property
+[NOTE]
+====
+This property is mandatory.
+====
+
+
+== Reset password workflow
+
+.URL to ask to change password
+
+It is possible to configure the path of your application where the end-user will be able to choose his new password. 
+
+[source, json]
+----
+{
+  "reset-password": {
+    "ask": {
+      "url": "http://localhost:3000/reset-password#${token}",     // <1>
+    }
+  }
+}
+----
+<1> The token is sent through the URL. To use it you need to put `${token}` in your URL.
+
+You can also only set the path of your url. The base url will be automatically provided by ZetaPush:
+
+[source, json]
+----
+{
+  "reset-password": {
+    "ask": {
+      "url": "/reset-password#${token}",     // <1>
+    }
+  }
+}
+----
+
+.Mandatory property
+[NOTE]
+====
+This property is mandatory.
+====
+
+
+.Email subject
+
+You can configure the subject of the sent email to reset the password:
+
+[source, json]
+----
+{
+  "reset-password": {
+    "ask": {
+      "email": {
+        "subject": "Please choose a new password"
+      }
+    }
+  }
+}
+----
+
+.Mandatory property
+[NOTE]
+====
+This property is mandatory.
+====
+
+
+== Email provider configuration
+
+To use the _StandardUserWorkflow_ you need to configure an email provider.
+Today, you can use two possibilities: _Mailjet_ or _SMTP_.
+
+.Mandatory property
+[NOTE]
+====
+You need to set at least one email provider configuration and you need to have only one email provider configuration enabled.
+====
+
+
+=== Mailjet configuration
+
+To use the _Mailjet_ configuration you need to configure the `application.json` file like this:
+
+[source, json]
+----
+{
+  "mailjet": {
+    "apikey-public": "my-mailjet-apikey-public",
+    "apikey-private": "my-mailjet-apikey-private",
+    "enable": "true",                                 // <1>
+    "url": "https://api.mailjet.com/v3.1/send"        // <2>
+  }
+}
+----
+<1> Optional property to define that we use this email provider configuration. (The default value is `true`)
```

Comment:
(The default value is `true` if you have configured apikey-public and apikey-private)